src/ykfde-disenroll

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# sanitize environment
+YKFDE_LUKS_DEV=""
+YKFDE_PRINT_ONLY=""
+LUKS_KEYSLOT=""
+DBG=""
+YKFDE_SLOT_CHECK=""
+YKFDE_CHALLENGE_SLOT="2"
+YKFDE_CHALLENGE_PASSWORD_NEEDED=""
+YKFDE_CHALLENGE=""
+YKFDE_RESPONSE=""
+YKFDE_PASSPHRASE=""
+
+if [ -r /etc/ykfde.conf ]; then
+  # shellcheck source=ykfde.conf
+  . /etc/ykfde.conf
+else
+  echo "WARNING: Can't access /etc/ykfde.conf. Falling back to defaults."
+fi
+
+while getopts ":d:s:vh" opt; do
+  case "$opt" in
+    d)
+      YKFDE_LUKS_DEV="$OPTARG"
+      printf '%s\n' "INFO: Setting device to '$OPTARG'."
+      ;;
+    s)
+      if [ "$OPTARG" -gt -8 ] && [ "$OPTARG" -lt 8 ]; then
+        LUKS_KEYSLOT="$OPTARG"
+        printf '%s\n' "INFO: Removing LUKS keyslot '$OPTARG'."
+      else
+        printf '%s\n' "ERROR: Chosen LUKS keyslot '$OPTARG' is invalid. Please choose valid LUKS keyslot number between '0-7'."
+        exit 1
+      fi
+      ;;
+    v)
+      DBG=1
+      echo "INFO: Debugging enabled"
+      ;;
+    h)
+      echo
+      echo " -d <device>   : select an existing device"
+      echo " -s <slot>     : kill this LUKS keyslot"
+      echo " -v            : show input/output in cleartext"
+      echo
+      exit 1
+      ;;
+    \?)
+      printf '%s\n' "Error: Invalid option: '-$OPTARG'" >&2
+      echo
+      echo " -d <device>   : select an existing device"
+      echo " -s <slot>     : kill this LUKS keyslot"
+      echo " -v            : show input/output in cleartext"
+      echo
+      exit 1
+      ;;
+  esac
+done
+
+YKFDE_SLOT_CHECK="$(ykinfo -q -"$YKFDE_CHALLENGE_SLOT")"
+[ "$DBG" ] && printf '%s\n' " > YubiKey slot status 'ykinfo -q -$YKFDE_CHALLENGE_SLOT': $YKFDE_SLOT_CHECK"
+
+if [ "$YKFDE_SLOT_CHECK" != 1 ]; then
+  printf '%s\n' "ERROR: Chosen YubiKey slot '$YKFDE_CHALLENGE_SLOT' isn't configured. Please choose slot configured for 'HMAC-SHA1 Challenge-Response' mode in '/etc/ykfde.conf'"
+  exit 1
+fi
+
+if [ -z "$YKFDE_PRINT_ONLY" ]; then
```

Comment:
`YKFDE_PRINT_ONLY` doesn't exist in [getopts](https://github.com/agherzan/yubikey-full-disk-encryption/pull/37/files#diff-a6a480de6f60c219e4236e53ce018f4dR24)